app/api/v2/maps.py

```diff
@@ -0,0 +1,75 @@
+""" bancho.py's v2 apis for interacting with maps """
+from __future__ import annotations
+
+from typing import Optional
+
+from fastapi import APIRouter
+from fastapi import status
+from fastapi.param_functions import Query
+
+from app.api.v2.common import responses
+from app.api.v2.common.responses import Success
+from app.api.v2.models.maps import Map
+from app.repositories import maps as maps_repo
+
+router = APIRouter()
+
+
+@router.get("/maps")
+async def get_maps(
+    server: str,
```

Comment:
this should be moved down to match db table; also should test if typing.Literal can/should be used here

Reply:
avoiding for now - there are other similar things happening elsewhere, might standardize these to use enums later.